src/Shared/Exceptions/RequestException.php

```diff
@@ -9,5 +9,13 @@
  */
 class RequestException extends Exception
 {
-
-}
\ No newline at end of file
+    /**
+     * @param string $message
+     * @param string|int $code
+     * @param \Throwable $previous
+     */
+    public function __construct($message = "", $code = 0, $previous = null)
+    {
+        parent::__construct($message, (int)$code, $previous);
```

Comment:
Sorry por marear, pero en base a lo que comentamos de que se construye ahora siempre con un `int`:

```suggestion
     * @param int $code
     * @param \Throwable $previous
     */
    public function __construct($message = "", $code = 0, $previous = null)
    {
        parent::__construct($message, $code, $previous);
```

Reply:
I'd like to keep it as it is now.. It looks more flexible and easy to work with

Reply:
I agree with @addUsername , PHP casting ensures safety with no cost :smile: